src/js/modules/supervisors/FileManager.ts

```diff
@@ -39,9 +42,11 @@ class FileManager {
   ) {
     try {
       this.watcher = new Inotify(this.submitDir);
+      this.removeWatcher = new Inotify(this.activeDir);
```

Comment:
From what I can see, it's not that there's a `watcher` and a `removeWatcher`. They don't watch specific events, but specific dirs.

Their name should be `submitDirWatcher` and `activeDirWatcher`.

Reply:
That's make sense

Reply:
Done